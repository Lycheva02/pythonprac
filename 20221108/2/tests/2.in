('a', 'esli'), ('ya', 'sdelayu'), ('vot', 'tak')
(0,0), (0,10), (0,10)

