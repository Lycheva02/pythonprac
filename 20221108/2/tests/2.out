Not a triangle
Not a triangle
